eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -610,4 +610,19 @@
   <!-- Suppress checks in azure-core-experimental -->
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck"
             files="com.azure.core.experimental.jsonpatch.(JsonPatchDocumentSerializer|JsonPatchOperationSerializer).java"/>
+
+
+  <!-- Suppress checks all azure-spring-cloud MissingJavadocMethod and MissingJavadocType -->
+  <suppress checks="MissingJavadocMethod" files=".*[/\\]com[/\\]azure[/\\]spring[/\\].*"/>
+  <suppress checks="MissingJavadocType" files=".*[/\\]com[/\\]azure[/\\]spring[/\\].*"/>
+  <suppress checks="JavadocPackage" files=".*com[/\\]azure[/\\]spring[/\\].*\.java"/>
```

Comment:
Why do we suppress this warning?

Reply:
Yes, I will remove the JavadocPackage suppress.